scilpy/reconst/frf.py

```diff
@@ -221,9 +222,10 @@ def compute_msmt_frf(data, bvals, bvecs, btens=None, data_dti=None,
         logging.warning('Your b-vectors do not seem normalized...')
         bvecs = normalize_bvecs(bvecs)
 
-    b0_thr = check_b0_threshold(force_b0_threshold, bvals.min(), bvals.min())
-
-    gtab = gradient_table(bvals, bvecs, btens=btens, b0_threshold=b0_thr)
+    # toDo. Using a fake b0_threshold here because currently, the
```

Comment:
I would keep the `check_b0_threshold` but with `tol` here. And pass `tol` to the gtab function. Also, maybe remove the toDo? :)

Reply:
It's still there but we the way it's implemented right now (passing args), it was easier to do the check in the script than in the function.